MANIFEST.in

```diff
@@ -6,6 +6,7 @@ recursive-include lib/iris/tests/results *.cml *.cdl *.txt *.xml *.json
 recursive-include lib/iris/etc *
 include lib/iris/fileformats/_pyke_rules/*.k?b
 include lib/iris/tests/stock*.npz
```

Comment:
Quite unrelated really, but I see that this line is now obsolete + should probably have been removed in #3913
```suggestion
```